vNext/channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -511,8 +511,11 @@ export class Sender implements IChannelControlsAI {
 
         if (queued) {
             this._buffer.markAsSent(payload);
+            // no response from beaconSender, clear buffer
+            this._onSuccess(payload, payload.length);
         } else {
-            this._logger.throwInternal(LoggingSeverity.CRITICAL, _InternalMessageId.TransmissionFailed, ". " + "Failed to send telemetry with Beacon API.");
+            this._logger.throwInternal(LoggingSeverity.WARNING, _InternalMessageId.TransmissionFailed, ". " + "Failed to send telemetry with Beacon API, retried with xhrSender.");
+            this._xhrSender(payload, true);
```

Comment:
We can now add (e2e) tests to make sure that it will correctly fallback here. Also add a test to verify it will use beacons when that mode is enabled.